src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,96 @@
+/* = Navigation
+============================ */
+.tribe-common {
+
+	.tribe-common-c-nav {
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+
+		ul {
+			display: block;
+			list-style: none;
+			width: 100%;
+		}
+	}
+
+	.tribe-common-c-nav__prev,
+	.tribe-common-c-nav__next {
+		display: inline-flex;
+		align-items: center;
+		color: var(--color-text-secondary);
+		font-size: var(--font-size-1);
```

Comment:
We're going to need `line-height` style for almost all text styling properties so that we properly set the size of the text.

Reply:
I think you're still missing this @juanfra , looks like the last one.

Reply:
I think you're still missing this last one @juanfra